_posts/2022-07-22-servo-kinematics.md

```diff
@@ -0,0 +1,31 @@
+---
+author: Wyatt Rees
+comments: false
+date: 2022-07-22 00:00:00 -0600
+layout: post
+slug: MoveIt-Servo-Inverse-Kinematics
+title: MoveIt Servo Inverse Kinematics Improvements
+media_type: gif
+media_link: /assets/images/blog_posts/ee_swirl.gif
+description: Inverse kinematics has been overhauled in MoveIt Servo
+
+categories:
+- MoveIt
+- ROS
+
+---
+
+A new, exciting improvement has made it into MoveIt! Servo! Until now, MoveIt! Servo has used the inverse Jacobian method for computing inverse kinematics. While this may work in many situations, a significant portion of robotics research has been dedicated to more performant algorithms for computing inverse kinematics. A primary focus of this research is _redundancy resolution_, in which robots with redundant degrees of freedom may utilize alternative joint configurations to achieve the same end-effector pose. By doing so, robots may be enabled to better avoid singularies, avoid collisions, and lower power consumption, to name a few benefits.
```

Comment:
Please make every sentence its own line. This helps reviewers because it is easier to comment on single sentences instead of whole paragraphs.

This is called semantic linefeeds. Reference: https://rhodesmill.org/brandon/2012/one-sentence-per-line/